synfig-studio/src/gui/app.cpp

```diff
@@ -1333,6 +1352,15 @@ App::get_default_accel_map()
 		{"<Control>l",              "<Actions>/canvasview/toggle-grid-snap"},
 		{"<Control>n",              "<Actions>/mainwindow/new"},
 		{"<Control>o",              "<Actions>/mainwindow/open"},
+		{"<Control>1",              "<Actions>/mainwindow/move-to-tab-1"},
```

Comment:
Use `<Primary>` instead of `<Control>`

Reply:
done and actually on windows it works same as control, which makes me wonder why all the "Control" are not just "Primary".